libs/ledgerjs/packages/hw-app-eth/tests/Eth.integration.test.ts

```diff
@@ -367,8 +553,10 @@ test("provideERC20TokenInformation", async () => {
   const zrxInfo = byContractAddressAndChainId(
     "0xe41d2489571d322189246dafa5ebde1f4699f498",
     1
+  )!;
```

Comment:
Since it's in a test file, I think the "t'inquiète 😉" is ok haha
If at some point the token doesn't exist, it's gonna be catched as an error either way while trying to get `zrxInfo.data` just after that and the test will fail. 
Other option would be to add a condition to check `zrxInfo` and throw if the token doesn't exist. I can change it this way if you prefer 👌

Reply:
If you ask me I can't think of a legit case to use this `!` operator, I would always prefer a safe check.
But definitely not a big deal in this case, it can stay that way up to you.